gc-cli/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	"presets": ["es2015"]
```

Comment:
I do not exactly know why, but I know that without them the tool does not work. It would complain about unexpected token `import` so I am assuming a that babel uses a `.babelrc` on a directory basis.